### PR TITLE
Support transfer reversals

### PR DESCRIPTION
**Pull request by danwang**
r? @bkrausz 

Mirrors refunds.

I'm unable to add any meaningful tests because the tests result in network requests, and there's no way to create a connected account for transferring.

### CODE REVIEW

File: init.php [hunk 1]
```diff
@@ -44,6 +44,7 @@
 require(dirname(__FILE__) . '/lib/Plan.php');
 require(dirname(__FILE__) . '/lib/Recipient.php');
 require(dirname(__FILE__) . '/lib/Refund.php');
+require(dirname(__FILE__) . '/lib/TransferReversal.php');
```
Comment (bkrausz) [inline init.php]: Alphabetical order please

File: lib/TransferReversal.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     * @param array|string|null $opts
+     *
+     * @return TransferReversal The saved reversal.
+     */
+    public function save($opts = null)
+    {
+        return $this->_save($opts);
+    }
+}
+
```
Comment (bkrausz) [inline lib/TransferReversal.php]: FYI Travis is complaining about the number of newlines at the end of this and the corresponding test file.

**Comment by bkrausz** [comment]
See nits, otherwise lgtm

File: init.php (hunk 1 above)
Comment (danwang) [inline init.php]: Fixed.

File: lib/TransferReversal.php (hunk 2 above)
Comment (danwang) [inline lib/TransferReversal.php]: Thanks! Fixed.

**Comment by bkrausz** [comment]
woot! :sheep:

**Pull request merged**